src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -520,10 +517,14 @@
 
 		<type fullname="System.Reflection.AssemblyName" preserve="fields" />
 
-		<type fullname="System.Reflection.EventInfo" preserve="fields">
-			<method name="AddEventFrame" />
-			<method name="StaticAddEventAdapterFrame" />
-		</type>
+    <!-- TODO: EventInfo has no fields in NetCore as it's an abstract class. Maybe we need RuntimeEventInfo fields
+         but those are already preserved below. -->
+		<!-- <type fullname="System.Reflection.EventInfo" > -->
+      <!-- TODO: There's a reference to these names in native code in security-core-clr.c around likne 735
```

Comment:
`security-core-clr.c` was used to implement the silverlight/moonlight security model ages ago, it is disabled in netcore.